webpack.config.js

```diff
@@ -52,6 +52,8 @@ var webpackConfig = {
     "news-node.behaviors":  path.resolve(srcJS,   "news-node.behaviors.js"),
     "news-node":            path.resolve(srcSass, "components/news-node/index.scss"),
     "news-card":            path.resolve(srcSass, "components/news-card/index.scss"),
+    "news-list-item":       path.resolve(srcSass, "components/news-list-item/index.scss"),
+    "news-list.behaviors":  path.resolve(srcJS,   "news-list.behaviors.js"),
     "newsletter":           path.resolve(srcSass, "components/newsletter/index.scss"),
     "news-assets":          path.resolve(srcSass, "components/newsletter/index.scss")
```

Comment:
Why are there two identical entry points? Can we remove one? It looks like news-assets is not used anywhere.